source/gui/addonStoreGui/controls/addonList.py

```diff
@@ -151,6 +151,7 @@ def OnColClick(self, evt: wx.ListEvent):
 		colIndex = evt.GetColumn()
 		log.debug(f"col clicked: {colIndex}")
 		self._addonsListVM.setSortField(self._addonsListVM.presentedFields[colIndex])
+		self.Parent.columnFilterCtrl.SetSelection(colIndex * 2)
```

Comment:
Does this work for both ascending and descending?

Reply:
Yes, done in cb0d08826019381b7b3228fa7e28c05eeffc303d.